docs/pages/versions/unversioned/sdk/app-integrity.mdx

```diff
@@ -0,0 +1,186 @@
+---
+title: App Integrity
+description: A library that provides access to Apple's App Attest service on iOS and Google's Play Integrity API on Android.
```

Comment:
```suggestion
description: A library that provides access to Google's Play Integrity API on Android and Apple's App Attest service on iOS.
```

Let's correct the platform order: https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#platform-order-when-referencing-android-ios-and-web-or-multiple-platforms.